cmd/devp2p/internal/ethtest/eth66_suite.go

```diff
@@ -483,8 +483,8 @@ func (s *Suite) TestNewPooledTxs_66(t *utesting.T) {
 				t.Fatalf("unexpected number of txs requested: wanted %d, got %d", len(hashes), len(msg))
 			}
 			return
-		case *NewPooledTransactionHashes:
-			// ignore propagated txs from old tests
+		case *NewPooledTransactionHashes, *NewBlock:
```

Comment:
I think it would also be nice to include `*NewBlockHashes` in there as well

https://travis-ci.com/github/ethereum/go-ethereum/jobs/501414461#L820

Reply:
Yup, fixed, thanks!